File: Dia 2/Go Lang/exercicio4.go
package main

import (
	"fmt"
)

func main() {
	var numero, somaPares, quantidadeImpares int
	var numerosPares []int
	var numerosImpares []int

	for i := 1; i <= 6; i++ {
		fmt.Printf("Digite o %dº número inteiro: ", i)
		fmt.Scan(&numero)

		if numero%2 == 0 {
			numerosPares = append(numerosPares, numero)
			somaPares += numero
		} else {
			numerosImpares = append(numerosImpares, numero)
			quantidadeImpares++
		}
	}

	fmt.Println("Números pares digitados:", numerosPares)
	fmt.Println("Soma dos números pares digitados:", somaPares)
	fmt.Println("Números ímpares digitados:", numerosImpares)
	fmt.Println("Quantidade de números ímpares digitados:", quantidadeImpares)
}

File: Dia 2/Python/exercicio4.py
# Faça um programa em Python que receba 6 numeros inteiros e mostre: ´
#  • Os numeros pares digitados; 
#  • A soma dos numeros pares digitados; 
#  • Os numeros   ımpares digitados;
#  • A quantidade de numeros  ımpares 

numeros_pares = []
numeros_impares = []

soma_pares = 0

for i in range(6):
    numero = int(input("Digite o {}º número inteiro: ".format(i + 1)))
    
    if numero % 2 == 0:
        numeros_pares.append(numero)
        soma_pares += numero
    else:
        numeros_impares.append(numero)


print("\nNúmeros pares digitados:", numeros_pares)
print("Soma dos números pares:", soma_pares)
print("Números ímpares digitados:", numeros_impares)
print("Quantidade de números ímpares:", len(numeros_impares))